tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigratorComparator.java

```diff
@@ -0,0 +1,73 @@
+package com.liferay.ide.project.ui.migration;
+
+import org.eclipse.jface.viewers.Viewer;
+import org.eclipse.jface.viewers.ViewerComparator;
+import org.eclipse.swt.SWT;
+
+/**
+ * @author Lovett li
+ */
+public class MigratorComparator extends ViewerComparator
```

Comment:
Load the format settings from our repository resources folder and then you can run format on the whole file to get things closer (ask Terry to see how)